api/models.py

```diff
@@ -0,0 +1,34 @@
+"""
+module models
+"""
+class Orders:
+    """class for orders"""
+    def __init__(self, order_id, client, contact, order_item, price, order_status):
+        """
+        constructor method for class
+        :param order_id:
+        :param client:
+        :param contact:
+        :param order_item:
+        :param price:
+        """
+        self.order_id = order_id
+        self.client = client
+        self.contact = contact
+        self.order_item = order_item 
+        self.price = price
+        self.order_status = order_status
+
+    def to_json(self):
+        """
+        method converts data to json format
+        :return: json_data"""
+        json_data = {
```

Comment:
Since you are setting all your attributes  on the Order class, you could keep your code `DRY` by leveraging  the `__dict__` attribute like this
```python
    def to_json(self):
        return self.__dict__
```

Reply:
I have read about the `__dict__` attribute and that's a good idea.